code/_textwork-style-landing.css

```diff
@@ -0,0 +1,63 @@
+@import url('https://fonts.googleapis.com/css2?family=Barlow+Condensed:wght@500&display=swap');
+
+
+body {
+    margin: 0;
+    padding-left: 70px;
+    color: #f0f0f0;
+    background-image: url(../img/home-typewriter-retro.jpeg);
+    background-size: cover;
+    background-repeat: no-repeat;
+    position: absolute;
+    overflow: hidden;
+    
+}
+
+p {
+    font-size: 25px;
+    text-transform: uppercase;
+    font-family: 'Barlow Condensed', sans-serif;
+}
+
+button {
+    background-color: transparent;
+    border: 2px solid #f0f0f0;
+    color: #f0f0f0;
+    text-align: center;
+    text-transform: capitalize;
+    padding: 10px 35px;
+    margin-right: 20px;
+    display: inline-block;
+    cursor: pointer;
+    font-size: 18px;
+    text-transform: uppercase;
+    transition-duration: 0.4s;
+    letter-spacing: 4px;
+    font-family: 'Barlow Condensed', sans-serif;
+}
+
+button:hover {
+    background-color: #f0f0f0;
+    color: #372719;
+}
+
+.logo {
+    padding-top: 200px;
+    padding-bottom: 10px;
+    width: 320px;
+}
+
+
+.oneline {
+    display: flex;
+    align-items: center;
+}
+
+.connect-logos-white a {
+    margin-left: 20px;
+    margin-right: 20px;

```

Comment:
A shorthand if you want to target both sides horizontally is to write: 
`margin: 0 20px;`
That simply means 0 margin on top/bottom, and 20px on left/right. You can also split it into 4 values:
`margin: 0 20px 0 20px;`
Which means 0 for top, 20px right, 0px bottom and 20px left. 4 values in clockwise order :)